README.md

```diff
@@ -54,7 +54,7 @@ If nothing is supplied the current working dir is used.
 Running monaco is done with required and non-required options and positional arguments:
 
 ```
-monaco --environments <path-to-environment-yaml-file> [--specific-environment <environment-name>] [--project <project-folder>] [--dry-run] [--verbose] [projects-root-folder]
+monaco --environments <path-to-environment-yaml-file> [--specific-environment <environment-name>] [--project <project-folder>] [--dry-run] [--verbose] [projects-root-folder] [--version]
```

Comment:
Can you put the `--version` before the [projects-root-folder]? The logic for extracting the folder currently depends on the fact that this is the last argument passed to `monaco`.

Reply:
I added a new example to show how to use the `-version` flag.

Reply:
Cool, right, you are also exiting the program when a `--version` is encountered. So, my comment is not valid anyway.